react_11/11-targil/text/src/counter.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useState } from 'react';
+
+export default function Counter(props) {
+  const { delta } = props;
+  const [count, setCount] = useState(0);
+
+  function inc() {
+    setCount(x => x + delta);
```

Comment:
No need for a function here, it can simply be `setCount(count + delta)`
Also, never use one-letter variables.